### PR TITLE
[Test infra] Enable manual trigger of e2e test

### PR DESCRIPTION
**Pull request by josephca**
## Description

To trigger e2e tests manually, an additional stage needs to be created. Currently, e2e tests can be triggered only through a PR.

## Which issue(s) does this PR fix

- Fixes https://issues.redhat.com/browse/RHIDP-1436

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Review by nickboldt — COMMENTED** [review]
feels like a no-op change here, so I would approve it but you have "WIP" on this PR...

**Comment by josephca** [comment]
> feels like a no-op change here, so I would approve it but you have "WIP" on this PR, and no linked JIRA.

I had "WIP" as it wasn't ready to be reviewed yet. :) Please review when you can. Thanks!

**Review by rnapoles-rh — APPROVED** [review]
/lgtm

**Review by nickboldt — APPROVED** [review]

File: .ibm/pipelines/functions.sh [hunk 1]
```diff
@@ -31,6 +31,12 @@ EOF
 
 skip_if_only() {
     echo "Checking if tests need to be executed..."
+    
+    if [[ ${SKIP_TEST,,} == "false" ]]; then
```
[the remark this answers is not in the document]
Comment (josephca) [reply .ibm/pipelines/functions.sh:35]: SKIP_TEST is added as an environment variable for the CI test pipeline for the manual test. It's not run via a GH action.

**Pull request merged**